src/utils/WellKnownUtils.ts

```diff
@@ -48,6 +56,14 @@ export function getE2EEWellKnown(): IE2EEWellKnown {
     return null;
 }
 
+export function getTileServerWellKnown(): ITileServerWellKnown {
+    const clientWellKnown = MatrixClientPeg.get().getClientWellKnown();
+    return (
+        clientWellKnown?.[TILE_SERVER_WK_KEY.name] ??
+        clientWellKnown?.[TILE_SERVER_WK_KEY.altName]
+    );
+}
+
```

Comment:
This can change over time, the js-sdk fires a `WellKnown.client` whenever it fetches a new set of directives, so this should respect that and not fall out of sync with the other parameters in that configuration.

Reply:
Thanks, I have addressed this in 6bb8d82feb